user-interface/src/data-verification/DataVerificationScreen.tsx

```diff
@@ -288,10 +292,18 @@ export default function DataVerificationScreen() {
                   <>
                     <div className="data-verification-accordion-header" data-testid="orders-header">
                       <div className="grid-row grid-gap-lg">
-                        <div className="grid-col-6 text-no-wrap">Court District</div>
-                        <div className="grid-col-2 text-no-wrap">Order Filed</div>
-                        <div className="grid-col-2 text-no-wrap">Event Type</div>
-                        <div className="grid-col-2 text-no-wrap">Event Status</div>
+                        <div className="grid-col-6 text-no-wrap">
+                          <h3>Court District</h3>
+                        </div>
+                        <div className="grid-col-2 text-no-wrap">
+                          <h3>Order Filed</h3>
+                        </div>
+                        <div className="grid-col-2 text-no-wrap">
+                          <h3>Event Type</h3>
+                        </div>
+                        <div className="grid-col-2 text-no-wrap">
+                          <h3>Event Status</h3>
+                        </div>
```

Comment:
```suggestion
                        <h3 className="grid-col-6 text-no-wrap">
                          Court District
                        </h3>
                        <h3 className="grid-col-2 text-no-wrap">
                          Order Filed
                        </h3>
                        <h3 className="grid-col-2 text-no-wrap">
                          Event Type
                        </h3>
                        <h3 className="grid-col-2 text-no-wrap">
                          Event Status
                        </h3>
```

Though it's unclear how the h3's will work with the grid.  Might need some css tweaks.